### ISSUE TITLE
Build with RN 0.58.3 and android SDK 28 fails

### ISSUE THREAD

**Issue by allan-cornet-eove** [mockingbot/react-native-zip-archive#117]
build release fails with RN 0.58.3 on android
```
ask :react-native-zip-archive:verifyReleaseResources FAILED
```
Due to old compileSdkVersion and buildToolsVersion versions

fix: 

replaces react-native-zip-archive/android/build.gradle by attached file
[build.gradle.zip](https://github.com/mockingbot/react-native-zip-archive/files/2816825/build.gradle.zip)

Standard way is to use  rootProject.ext.compileSdkVersion and rootProject.ext.buildToolsVersion

**Comment by sudovijay** [comment]
Same here - RN 0.57.8

`To suppress this warning, remove "buildToolsVersion '26.0.1'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.`

**Comment by allan-cornet-eove** [comment]
> Same here - RN 0.57.8
> 
> `To suppress this warning, remove "buildToolsVersion '26.0.1'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.`

With 0.58.3 it is not a warning, it is an blocking error to use.

Please notice that build.gradle.zip attached contains the best fix.
It would be appreciated that it be integrated quickly and package in npm.

**Comment by jonathangreco** [comment]
@plrthink Hi we really need this one, is it possible to have this fix soon ?

**Comment by jonathangreco** [comment]
@plrthink any news on this ? Is something wrong ?

**Comment by plrthink** [comment]
should be resolved with releasing 3.0.2
